packages/php-wasm/compile/php/phpwasm-emscripten-library-file-locking-for-node.js

```diff
@@ -754,34 +754,37 @@ const LibraryForFileLocking = {
 #if ASYNCIFY == 2
 		return Asyncify.handleAsync(async () => {
 #endif
+			// We have to get the VFS path from the file descriptor
+			// before closing it.
+			const [vfsPath, vfsPathResolutionErrno] =
```

Comment:
Aha, good spot

Reply:
> Aha, good spot

Thanks! Confession time: The truth is that it was correct with the initial multi-worker merge. Then I made an incorrect "fix". And this just restores the original good behavior.